content/blog/posts/open-source-bridge-promotes-community.rst

```diff
@@ -1,52 +1,89 @@
 Open Source Bridge Promotes Community
 #####################################
```

Comment:
Lets be consistent, either all ===== or all ###### for top level headers (i think === is more standard)